notebooks/gauss_plot.ipynb

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Set this to False manually if running this notebook locally in stand-alone mode (without cloning the repo).
</pre><p>No need for these lines now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>